BeardedManStudios/Source/Forge/Networking/UDPClient.cs

```diff
@@ -285,6 +285,35 @@ private void ReadNetwork()
 							// Ping the server to finalize the player's connection
 							Send(Text.CreateFromString(Time.Timestep, InstanceGuid.ToString(), false, Receivers.Server, MessageGroupIds.NETWORK_ID_REQUEST, false), true);
 						}
+						else if (packet.Size > 17)
```

Comment:
Tiny nitpick, could this 17 be moved to a constant? It's referenced slightly later in the file too